nodestream/schema/printers/graph_schema_extraction.py

```diff
@@ -0,0 +1,25 @@
+from .graphql_schema_printer import GraphQLSchemaPrinter
```

Comment:
Can you add the `Neo4jLLMGraphSchemaExtraction` to the package level `__init__.py` so that its loaded and available?